### ISSUE TITLE
Possibility to disable all builtin highlights

### ISSUE THREAD

**Issue by david-hill** [hexchat/hexchat#2151]
https://github.com/david-hill/irc/blob/master/hexchat-2.12.4-disable-hilight-ng.patch

**Comment by TingPing** [comment]
You don't need to disable highlights, I'm also confused why you combined my two scripts that do independent things, it just increases the complexity of me seeing what your real problem is.

**Comment by TingPing** [comment]
Oh Ok I get it, You want to handle "cases" from highlights too, just add another hook that doesn't re-emit the message but still adds it to a tab.

**Comment by david-hill** [comment]
I'd need to retest this ... Now I'm running with my patch and everything's fine.

**Comment by david-hill** [comment]
If I do that, the builtin notification will still create a problem ... Because if I don't emit_print that notification, it won't appear in the chat window.   Does this make sense?

**Comment by TingPing** [comment]
> If I do that, the builtin notification will still create a problem ... Because if I don't emit_print that notification, it won't appear in the chat window. Does this make sense?

No. If you receive a highlight event you just.. don't emit anything different.
